modules/ui/README.md

```diff
@@ -38,7 +38,7 @@ Install Yo, Grunt, Bower, and Compass
 If you want to test how the UI runs from the combined and minified files (which you would generally use
 when you go to Production), you will need to do the following (after you have done the build steps above):
 
-Edit constants.json with the following change to the apiHostBaseUrlValue value, since you would be running the
+Edit default_constants.json with the following change to the apiHostBaseUrlValue value, since you would be running the
```

Comment:
Ok, are you referring to the root level README in the project? Do I need to make another change somewhere else?